website/docs/r/automation_schedule.html.markdown

```diff
@@ -61,6 +65,26 @@ The following arguments are supported:
 
 * `timezone` - (Optional) The timezone of the start time. Defaults to `UTC`. For possible values see: https://msdn.microsoft.com/en-us/library/ms912391(v=winembedded.11).aspx
 
+* `advanced_schedule` - (Optional) Advanced fine-grained schedule frequency configuration. The `advanced_schedule` block supports fields documented below.
+
+---
+
+The `advanced_schedule` block supports:
+
+* `week_days` - (Optional) List of days of the week that the job should execute on.
+
+* `month_days` - (Optional) List of days of the month that the job should execute on. Must be between 1 and 31. 0 for last day of the month.
```

Comment:
>0 [](start = 112, length = 1)

*minor* can we quote the numbers in backtick such that " Must be between `1` and `31`".
*question* are you sure `0` represents the last day of the month, or `-1`?

Reply:
It's indeed `-1`. Thank you for the correction.